### PR TITLE
chore(package): explicit usage of `uglifyjs v2.0.0`

### PR DESCRIPTION
**Pull request by pierreneter**

### CODE REVIEW

**Comment by sandfox** [comment]
I'm pretty sure that `^2.8.2` in semver already excludes `3.0.0.` and beyond.

https://jubianchi.github.io/semver-check/ has what I believe is an accurate tester.

**Comment by pierreneter** [comment]
Opps. My fault, i think this will update to `uglify-js 3` when i run `npm update` but it not. Still `^2.8.29` is enough.
And one more, `uglify-js` required but did't install when i run `npm install uglifyjs-webpack-plugin --save` i think we need `uglify-js` in both `dependencies` and `peerDependencies`

**Comment by pierreneter** [comment]
screen shot 2017-06-23 at 1 38 20 pm

File: package.json [hunk 1]
```diff
@@ -42,11 +42,12 @@
   },
   "homepage": "https://github.com/webpack-contrib/uglifyjs-webpack-plugin",
   "peerDependencies": {
-    "webpack": "^1.9 || ^2 || ^2.1.0-beta || ^2.2.0-rc || ^3.0.0"
+    "webpack": "^1.9 || ^2 || ^2.1.0-beta || ^2.2.0-rc || ^3.0.0",
+		"uglify-js": "^2.8.29"
```
Comment (evilebottnawi) [inline package.json]: `uglify-js` already in dev deps, why your need peerDeps? This is unnecessary

Comment (pierreneter) [inline package.json]: let me show you why.

**Comment by michael-ciniawsky** [comment]
@pierreneter Did you clean the cache and tried a 'fresh' install ? As others already stated the `semver` range is correct, but if there are still regressions we need to find a solution :)

**Comment by pierreneter** [comment]
i think my problem is realated #52, or I have a big misunderstanding, but my project only work after i installed `uglify-js` at root of project. Maybe `uglify-js` in `dependencies` is redundant. I think my problem shall be done here. We can discuss about it in #52 or #49. Sorry for disturbing your time

**Comment by michael-ciniawsky** [comment]
Alright closing in favour of #52 then :)

**Pull request closed without merging**